components/Tabs/Tabs.js

```diff
@@ -41,18 +47,19 @@ class TabLink {
 class TabCard {
   constructor(element){
     // Assign this.element to the passed in element.
-    this.element;
+    this.element = tabs;
   }
   selectCard(){
     // Update the style of this.element to display = null
-    this.element;
+    this.element.display = null;
   }
 
 }
 
 // Create a reference to all ".tab" classes
-let tabs = document.querySelectorAll();
+let tabs = document.querySelectorAll(".tab");
 // Map over the array and convert each tab reference into a new TabLink object.  Pass in the tab object to the Tabs class.
-tabs = Array.from(tabs).map()
+tabs = Array.from(tabs).map(tab => new TabLink(tab));
 
-//Once you are complete, call the .select method on the first tab
\ No newline at end of file
+//Once you are complete, call the .select method on the first tab
+cards[0].selectCard();
```

Comment:
tabs, you want to call .select (and actually .selectTab, that was a typo), on tabs from line 62. `tabs[0].selectTabs()`